### ISSUE TITLE
Please add support for _.pluck

### ISSUE THREAD

**Issue by cheapsteak** [tandrewnichols/lodash-aliases#2]
Thanks!

**Comment by tandrewnichols** [comment]
Oh . . . I wonder how I missed that. Probably because I actually like the new `_.map(list, 'prop')` thing that replaced it. I'll try to get that added soon.

**Comment by tandrewnichols** [comment]
Oh, I remember why I didn't add this. Pluck was never an alias. It had it's own functionality that was later merged into `_.map`. Is there a reason you can't use `_.map` for what you're trying to do? Just have `_.pluck` all over and don't want to update it (which is a legit reason and the main one I wrote this to address originally)?

**Comment by cheapsteak** [comment]
Yup, 2nd time finding myself working on a project with chunks of legacy code that we'd prefer not to take ownership of. Currently I'm dropping this library in then adding `_.pluck = _.map` :)

**Comment by tandrewnichols** [comment]
Ok. I've been thinking about it . . . it feels like a slightly weird thing to do since it will make `_.pluck` work exactly like `_.map`, but I think I'll still do it. Not sure if I'll do it like that though. I have one other idea I'll try first.

**Comment by tandrewnichols** [comment]
So . . .

I actually created a differently library, [lodash-backports](https://github.com/tandrewnichols/lodash-backports), and will be deprecating this one. I think that more clearly implies that I'm supporting other deprecated functions and not just aliases. I just published version 1.0.0 of that one. Let me know if that solves your issue, or maybe open a new issue there if not.

Note that the usage of the library did change a bit (I worked in some other changes/improvements I'd been intending to do anyway), so have a look at the README to get up to speed.
